CMSgov/mint-app
# [NOREF] update job codes

# NOREF

## Changes and Description

- Update job codes to be the ones we're using in our lower level environments

## How to test this change

- Ensure you have the job codes `MINT_USER_NONPROD` and `MINT_ASSESSMENT_NONPROD`
- Attempt to log in locally through Okta
- Ensure that you can access and use the application

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
